mesg/deploy_deployer.go

```diff
@@ -113,7 +114,12 @@ func (d *serviceDeployer) deploy(path string) (*service.Service, *importer.Valid
 	if err != nil {
 		return nil, nil, err
 	}
-	d.sendStatus(fmt.Sprintf("%s Image built with success.", aurora.Green("✔")), DONE)
+	if _, err := os.Stat(filepath.Join(path, ".mesgignore")); err == nil {
+		// TODO: remove for a future release
+		d.sendStatus(fmt.Sprintf("%s [DEPRECATION] please use .dockerignore instead of .mesgignore", aurora.Red("⨯")), DONE)
+	} else {
```

Comment:
Remove else? there will be warning about .mesgingore file, but there will be no info about build complete status (which will happen here). So display deprecation msg + build msg.

Todo: s/DEPRECATION/DEPRECATED/

Reply:
i agree, requesting changes